staging/src/kubevirt.io/client-go/log/log.go

```diff
@@ -333,6 +333,7 @@ func (l FilteredLogger) Errorf(msg string, args ...interface{}) {
 
 func (l FilteredLogger) Critical(msg string) {
 	l.Level(FATAL).msg(msg)
+	panic(msg)
```

Comment:
Oh interesting, i guess this answers my question earlier. However i remember seeing another place where there were Critical() logs with panics right under them. It looks inconsistent.

Are we sure we want to do this? Changing the behavior here seems scary.

Reply:
Every `Critical` call in the code had a `panic` afterwards and I wanted to make it similar to `log.Fatalf`. I guess I should at least also do that for `Criticalf`. Maybe also rename `Critical` to `Fatal`?

Reply:
i have no idea what critical is supposed to even mean compared to fatal and error. If something is critical i'd think it's unrecoverable which is the same as fatal. 

Can we remove critical and just use Error and Fatal to replace ?